compiler/noirc_driver/tests/stdlib_warnings.rs

```diff
@@ -3,6 +3,7 @@ use std::path::Path;
 use noirc_driver::{file_manager_with_stdlib, prepare_crate, ErrorsAndWarnings};
 use noirc_frontend::hir::{def_map::parse_file, Context};
 
+#[ignore = "Temporarily ignoring the test until the stdlib is updated to use explicit numeric generics"]
```

Comment:
Had to ignore now that we default the stdlib. Making a follow-up to update the stdlib as to not clutter this PR any further.